htsinfer/get_library_source.py

```diff
@@ -254,10 +254,10 @@ def get_source_expression(
 
         Returns:
             Data frame with columns `source_ids` (a tuple of source short name
-                and taxon identifier, e.g., `("hsapiens", 9606)`) and `tpm`,
-                signifying the percentages of total expression per read source.
-                The data frame is sorted by total expression in descending
-                order.
+                and taxon identifier, e.g., `("hsapiens", 9606)`) and
+                `est_counts`, signifying the percentages of total expression
+                per read source. The data frame is sorted by total expression
```

Comment:
I'm not sure it is accurate to speak of "expression" when we use counts. Perhaps best to refer to "total estimated counts".

Reply:
Thanks, addressed it in commit